### ISSUE TITLE
CFG公式为什么和图像diffusion中的不同？

### ISSUE THREAD

**Issue by webro12138** [ML-GSAI/LLaDA#104]
logits = un_logits + (cfg_scale + 1) * (logits - un_logits)
按图像diffusion中CFG应该是：
  logits = un_logits + cfg_scale * (logits - un_logits)
